flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraSinkBaseTest.java

```diff
@@ -361,6 +381,12 @@ private TestCassandraSink createOpenedSendExceptionTestCassandraSink(CassandraSi
 		return testCassandraSink;
 	}
 
+	private TestCassandraSink createOpenedSendErrorTestCassandraSink(CassandraSinkBaseConfig config) {
```

Comment:
can be static

Reply:
Done; for consistency I also made the ExceptionSink static.